packages/EasyErrorHandler/docs/config.md

```diff
@@ -47,13 +47,14 @@ Laravel has the following additional configuration option:
 
 Symfony has the following additional configuration options:
 
-| Configuration                      | Default    | Description                                                                                              |
-|------------------------------------|------------|----------------------------------------------------------------------------------------------------------|
-| `bugsnag_ignore_validation_errors` | `true`     | Ignore validation errors based on ApiPlatformValidationErrorsResponseBuilder                             |
-| `override_api_platform_listener`   | `true`     | If using [API Platform][1], override its inbuilt exception handling to use the EasyErrorHandler package. |
-| `transform_validation_errors`      | `true`     | Transform validation errors response structure based on ApiPlatformValidationErrrorResponseBuilder.      |
-| `translation_domain`               | `messages` | Symfony translation domain.                                                                              |
-| `verbose`                          | `false`    | Use extended error response containing exception message, trace, etc.                                    |
+| Configuration                      | Default    | Description                                                                                                         |
+|------------------------------------|------------|---------------------------------------------------------------------------------------------------------------------|
+| `bugsnag_ignore_validation_errors` | `true`     | Ignore validation errors based on ApiPlatformValidationErrorsResponseBuilder                                        |
+| `error_codes_enums_auto_discovery` | `false`    | Try recursively to find all enums with error codes that implement ErrorCodesEnumInterface in the project directory. |
```

Comment:
Please fix the description if we base our logic on the `AsErrorCodes` attribute like so `Find all error code enums having AsErrorCode attribute in the project directory recursively.`.

Reply:
And do we really need this option? We use `AsErrorCodes` for the only purpose only - to allow EasyErrorHandler collect all the implementations. So if you use this attribute the package can automatically collect enums.

Reply:
error_codes_enums_auto_discovery removed